src/components/atoms/SearchInfoPopup/SearchInfoPopup.jsx

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react';
+import { Card, CardContent, IconButton, Typography } from '@mui/material';
+import Zoom from '@mui/material/Zoom';
+import CloseRoundedIcon from '@mui/icons-material/CloseRounded';
+
+export default function SearchInfoPopup() {
+  const [popupOpen, setPopupOpen] = useState(true);
+
+  const handlePopupToggle = () => {
+    setPopupOpen(!popupOpen);
```

Comment:
When setting state based off of previous state it is best to use a callback with the previous state as an argument. 
`setPopUpOpen((prev) => !prev);`

This will ensure that state is maintained correctly between renders.